brownie/network/account.py

```diff
@@ -944,26 +944,24 @@ def _apply_fee_to_tx(
 ):
     tx = tx.copy()
 
-    if gas_price is not None and (max_fee or priority_fee):
-        raise ValueError("gas_price and (max_fee, priority_fee) are mutually exclusive")
+    if gas_price is not None:
+        if max_fee or priority_fee:
+            raise ValueError("gas_price and (max_fee, priority_fee) are mutually exclusive")
+        tx["gasPrice"] = web3.toHex(gas_price)
```

Comment:
Check if toHex is really required here, I think web3.py does them automatically now.